project_name/home/tasks.py

```diff
@@ -0,0 +1,25 @@
+"""
+Sample tasks file.
+Will fallback to a simple task decorator that returns the original function.
+"""
+
+# The following will check to see if we can import task from celery -
+# if not then we definitely haven't installed it.
+# Taken from wagtail core
+try:
+    from celery.decorators import task
```

Comment:
As this is in requirements.txt, this test will always succeed even if celery is not actually running.

I don't think it's useful to test for celery as the developer should be aware if their project uses celery or not.